lib/engine/game/g_18_al.rb

```diff
@@ -17,17 +17,24 @@ class G18AL < Base
       GAME_DESIGNER = 'Mark Derrick'
       GAME_END_CHECK = { bankrupt: :immediate, stock_market: :current_or, bank: :current_or }.freeze
 
+      EVENTS_TEXT = Base::EVENTS_TEXT.merge('remove_tokens' => ['Remove Tokens', 'Coal Field token removed']).freeze
+
+      ROUTE_BONUSES = %i[route_bonus_Atlanta_Birmingham route_bonus_Mobile_Nashville].freeze
```

Comment:
why the funny names? why not just atlanta_birmingham and mobile_nashville

Reply:
Probably because they existed as an attribute route_bonus before....
By updated now to the shorter names.